### PR TITLE
Extract common Blacklight::Component base class for blacklight components 

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]

**Comment by jcoyne** [comment]
It looks like there's an upstream fix for this: https://github.com/github/view_component/pull/1317  Should we wait it out?

**Comment by jcoyne** [comment]
The fix is released now: https://github.com/github/view_component/blob/main/docs/CHANGELOG.md#2520

File: lib/blacklight/component.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Blacklight
+  class Component < ViewComponent::Base
+    def render_in(view_context, &block)
```
Comment (jcoyne) [inline lib/blacklight/component.rb:5]: Can you remove this?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: lib/blacklight/component.rb (hunk 1 above)
Comment (cbeer) [reply lib/blacklight/component.rb]: Done

**Review by jcoyne — APPROVED** [review]

**Pull request merged**